code/game/jobs/job/science_vr.dm

```diff
@@ -29,4 +29,27 @@
 
 /datum/job/roboticist
 	total_positions = 3
-	pto_type = PTO_SCIENCE
\ No newline at end of file
+	pto_type = PTO_SCIENCE
+
+//////////////////////////////////
+//			Xenobotanist
+//////////////////////////////////
+/datum/job/xenobotanist
+	title = "Xenobotanist"
+	flag = XENOBOTANIST
+	departments = list(DEPARTMENT_RESEARCH)
+	department_flag = MEDSCI
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "the Research Director"
+	selection_color = "#633D63"
+	economic_modifier = 7
+	access = list(access_robotics, access_tox, access_tox_storage, access_research, access_xenobotany, access_hydroponics)
+	minimal_access = list(access_research, access_xenobotany, access_hydroponics, access_tox_storage)
+
+	minimal_player_age = 14
+
+	outfit_type = /decl/hierarchy/outfit/job/science/xenobiologist
+	job_description = "A Xenobotanist grows and cares for a variety of abnormal, custom made, and frequently dangerous plant life. When the products of these plants \
+					is both safe and beneficial to the station, they may choose to introduce it to the rest of the crew."
```

Comment:
> When the products of these plants **is** both safe and beneficial to the station

Grammar. I believe 'are' was intended?

Reply:
You would be correct.

Reply:
Funny that you caught that, that's the original writing from the alt title.

Reply:
Fixed this.